tests/test_update.py

```diff
@@ -192,13 +193,15 @@ def test_network_based_image_update(
             s3_address,
         )
 
-        output = connection.run("fw_printenv bootcount").stdout
+        bootenv_print, bootenv_set = bootenv_tools(connection)
```

Comment:
`bootenv_set` here is unused
```suggestion
        bootenv_print, _ = bootenv_tools(connection)
```

Reply:
Thanks, fixed!

Reply:
Somehow you re-introduced this in your last force-push.

Reply:
Wah, how did that happen? Did I use an old branch? I will have to compare with the SHAs in this PR now, I don't trust that I didn't mess up something else too.

Reply:
This should be fixed now, and I verified to be the best of my ability that I didn't miss anything else in the force push.